pypeit/reduce.py

```diff
@@ -816,8 +1014,7 @@ def local_skysub_extract(self, waveimg, global_sky, sobjs,
         return self.skymodel, self.objmodel, self.ivarmodel, self.outmask, self.sobjs
 
 
```

Comment:
Can we replace this with get_instance as in my new classes?

Reply:
I ask that you do that in your branch.